### ISSUE TITLE
Windows 11 22621 

### ISSUE THREAD

**Issue by jiangnancao** [sylveon/EarlyStart#4]
Version Windows 11 Professional Workstation Edition

Version 22H2

Installation date May 21, 2023

Operating system version 22621.1928

Experience Windows Feature Experience Pack 1000.22644.1000.0

If it fails, please fix the source code

**Comment by sylveon** [comment]
Does it not work?

**Comment by jiangnancao** [comment]
> Does it not work?

Yes, the service can be started, but the added program did not start

**Comment by Dhyfer1** [comment]
@sylveon The same thing happens to me in the latest Dev build of Windows 11. I use earlystart to start translucenttb (portable version) before the other programs that run at startup, but after multiple reboots translucenttb takes a long time to start even if I don't use earlystart.

I also have a question.  Earlystart starts the programs according to the order in which they are listed?

**Comment by which-is-my-way** [comment]
Me,too.

**Comment by sylveon** [comment]
I found the issue, it has to do with this setting:
![image](https://github.com/sylveon/EarlyStart/assets/6440374/79f25008-136c-4d9b-9335-cd81bfec417c)

If this is on, your session opens before EarlyStart starts. I'm working on an update that will make EarlyStart launch programs in each session that is currently opened when it starts.

> Does Earlystart launch these three programs in order or does it just grab any program in the list and run it first?

It runs them in the order that you specify in the .earlystart file.

> If a program X creates a shortcut in the startup folder (in shell:startup), and if I use earlystart to start program X, can I safely delete the shortcut created in the startup folder?

Yes, because EarlyStart starts the program via other means, you can remove that shortcut.

**Comment by Dhyfer1** [comment]
> I found the issue, it has to do with this setting:
> ![image](https://github.com/sylveon/EarlyStart/assets/6440374/79f25008-136c-4d9b-9335-cd81bfec417c)
> 
> If this is on, your session opens before EarlyStart starts. I'm working on an update that will make EarlyStart launch programs in each session that is currently opened when it starts.

Does that image mean that the next earlystart update comes with a graphical interface? 👀 It's good to know that earlystart is getting another update, since the last update was 5 years ago, and thank you very much for answering my questions :)

**Comment by sylveon** [comment]
No, that screenshot is from windows settings

**Comment by sylveon** [comment]
Yes, it should work if turned off

**Comment by Dhyfer1** [comment]
> Yes, it should work if turned off

Ok. I have turned it off and it works, except for translucenttb. I have added the executable path of the portable version of translucenttb in the .earlystart file, but after restarting my PC and logging in I am facing this error.  How to solve it? 

![image_2 (1).png](https://github.com/sylveon/EarlyStart/assets/50301201/c5a1ed9b-b64a-432f-9ca4-449c8a30c13b)

I have two other programs listed in the earlystart file, and these two programs run without problem.  But I can't get translucenttb to work when I use it together with earlystart.

**Comment by sylveon** [comment]
TTB will need some small updates to work correctly with EarlyStart. Future versions of EarlyStart will also automatically detect that TTB from the store is installed and launch it.

**Comment by Dhyfer1** [comment]
That's fine, but as long as translucenttb or earlystart receive their next update, how can I solve the error that I showed you. 

The portable version of translucenttb does not have any configuration to run at system startup, so I have to add a shortcut to the executable in the startup folder (as the translucenttb readme says) and this way translucenttb runs without problem when it is not listed in the earlystart file, but every time I restart my PC translucenttb takes a while to start, that's why I am using it together with earlystart.  But you know the error I get when translucenttb is listed

**Comment by brownjeesus** [comment]
> TTB will need some small updates to work correctly with EarlyStart. Future versions of EarlyStart will also automatically detect that TTB from the store is installed and launch it.

Well is it dont yet, cause it's raising the same issue with the xaml diagonistics, I'm on windows 11.

**Comment by Ptilopsis01** [comment]
I turned the windows option off, but it still do not work.
(Windows 11 22631.2861)
